OPHD/States/CrimeRateUpdate.cpp

```diff
@@ -27,6 +38,13 @@ void CrimeRateUpdate::update(const std::vector<TileList>& policeOverlays)
 		int crimeRateChange = isProtectedByPolice(policeOverlays, structure) ? -1 : 1;
 		structure->increaseCrimeRate(crimeRateChange);
 
+		// Crime Rate of 0% means no crime
+		// Crime Rate of 100% means crime occurs 10% of the time
+		if (structure->crimeRate() + random() > 1000)
```

Comment:
Some magic numbers going on here that aren't documented -- not super picky about it but something to be aware of.

This is probably a good place to have difficulty modifiers (e.g., easy might be 5%, hard might be 20%) though this is probably beyond the scope of this PR and change set.

Reply:
Yeah, agree on the difficulty modifier being a task here. I'll add it to the umbrella Implement Crime task.